lition/adapter.js

```diff
@@ -0,0 +1,33 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+
+const customError = (data) => {
+  if (data.Response === 'Error') return true
+  return false
+}
+
+const customParams = {
+  source: ['source'],
+}
+
+const execute = (input, callback) => {
+  const validator = new Validator(input, customParams)
+  if (validator.error) return callback(validator.error.statusCode, validator.error)
+
+  const jobRunID = validator.validated.id
+  const source = validator.validated.data.source
+
+  const url = `https://staking.lition.io/api/v1/energy/source/${source}/date/2020-10-07/hour/15/`
```

Comment:
Why is the date part of the endpoint`date/2020-10-07/hour/15/` fixed?
How should this be used, and would it make sense to take this as a param with sensible defaults?

Reply:
That's a good point. I'll go to Devlin and get more details on how they want this used.

Reply:
I've modified it to be a little more flexible with these.

One thing I am unsure of is what timezone they are basing their API off of, I presume UTC. I am able to query a full 24 hours ahead. I'll talk to Johann again tomorrow before merging this.